kubernetes/kfserving/README.md

```diff
@@ -231,7 +237,25 @@ The response is as below :
   ]
 }
 ```
-For the request and response of BERT and Text Classifier models, refer the "Request and Response" section of section of [BERT Readme file](https://github.com/pytorch/serve/blob/master/kubernetes/kfserving/Huggingface_readme.md) and [Text Classifier Readme file](https://github.com/pytorch/serve/blob/master/kubernetes/kfserving/text_classifier_readme.md). .
+
+KFServing supports Static batching by adding new examples in the instances key of the request:
```

Comment:
Somewhere, we should add that when performing static batching this way, the user needs to make sure they're using batch size of 1 (even though more than one input instance is specified in the input). Doing otherwise will cause a failure.

Reply:
@fbbradheintz Added it